src/stravalib/protocol.py

```diff
@@ -162,8 +162,12 @@ def authorization_url(
         )
 
     def exchange_code_for_token(
-        self, client_id: int, client_secret: str, code: str
-    ) -> AccessInfo:
+        self,
+        client_id: int,
+        client_secret: str,
+        code: str,
+        return_athlete: bool | None = False,
+    ) -> AccessInfo | tuple[AccessInfo, dict[str, Any]]:
```

Comment:
```suggestion
    ) -> AccessInfo | tuple[AccessInfo, dict[str, Any] | None]:
```
I forgot that the athlete may be None